cmd/cstor-pool-mgmt/controller/replica-controller/new_replica_controller_test.go

```diff
@@ -48,3 +52,34 @@ func TestNewCStorVolumeReplicaController(t *testing.T) {
 		t.Fatalf("Pool controller object's OpenebsClientset mismatch")
 	}
 }
+
+func TestEnqueueCStorReplica(t *testing.T) {
+	fakeKubeClient := fake.NewSimpleClientset()
+	fakeOpenebsClient := openebsFakeClientset.NewSimpleClientset()
+
+	kubeInformerFactory := kubeinformers.NewSharedInformerFactory(fakeKubeClient, time.Second*30)
+	openebsInformerFactory := informers.NewSharedInformerFactory(fakeOpenebsClient, time.Second*30)
+
+	// Instantiate the cStor VolumeReplica controllers.
+	volumeReplicaController := NewCStorVolumeReplicaController(fakeKubeClient, fakeOpenebsClient, kubeInformerFactory,
+		openebsInformerFactory)
+	test := apis.CStorVolumeReplica{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: "VolumeReplicaResource1",
+			UID:  "abcd123",
+			Labels: map[string]string{
+				"cstorpool.openebs.io/uid":  "123abc",
+				"cstorpool.openebs.io/name": "cstor-123abc",
+			},
+		},
+		Spec: apis.CStorVolumeReplicaSpec{
+			TargetIP: "127.0.0.1",
+			Capacity: "100MB",
+		},
+	}
+	q := common.QueueLoad{}
+	volumeReplicaController.enqueueCStorReplica(&test, q)
```

Comment:
Good effort @farhaanbukhsh 
Ideally speaking we should not test this function as it in turn uses all the inbuilt library functions.
Additionally, pushing into queue won't reflect size immediately as it is a rate limited and non blocking call.
I cannot think of any cases where we might need to test the function.

Reply:
ahh okay so what should I be testing :disappointed: ?